From: David Paleino <dapal@debian.org>
Subject: fix typos in binary
Forwarded: no

---
 mainwindow.cpp |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

--- diffpdf.orig/mainwindow.cpp
+++ diffpdf/mainwindow.cpp
@@ -166,12 +166,12 @@ void MainWindow::createWidgets(const QSt
     page1Label = new QLabel;
     page1Label->setAlignment(Qt::AlignCenter);
     page1Label->setToolTip(tr("<p>Shows the first (left hand) document's "
-                "page that correponds to the page shown in the "
+                "page that corresponds to the page shown in the "
                 "View Difference combobox."));
     page2Label = new QLabel;
     page2Label->setAlignment(Qt::AlignCenter);
     page2Label->setToolTip(tr("<p>Shows the second (right hand) "
-                "document's page that correponds to the page shown in "
+                "document's page that corresponds to the page shown in "
                 "the View Difference combobox."));
     logEdit = new QPlainTextEdit;
 
