From: David Paleino <dapal@debian.org>
Date: Thu, 29 Dec 2011 18:05:00 +0100
Subject: fix lintian warning, hyphen-used-as-minus-sign

Origin: vendor
Forwarded: no
---
 diffpdf.1.cmake | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/diffpdf.1.cmake b/diffpdf.1.cmake
index dcf3739..8f7cf31 100644
--- a/diffpdf.1.cmake
+++ b/diffpdf.1.cmake
@@ -25,7 +25,7 @@ second. This will make DiffPDF compare pages in the pairs (1, 1), (2,
 .PP
 If the program is passed two PDF filenames on the command line it will
 start up and compare those files in Text mode (or in Appearance mode if
-the filenames are preceded by -a or --appearance).
+the filenames are preceded by \-a or \-\-appearance).
 .SH "SEE ALSO"
 For a command line tool for comparing PDF files see
 http://www.qtrac.eu/comparepdf.html.
