From: Johannes Schauer <j.schauer@email.de>
Date: Mon, 2 Feb 2015 11:23:30 +0100
Subject: do not require the file extension to be pdf

---
 main.cpp | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/main.cpp b/main.cpp
index 7a6566d..94c2a25 100644
--- a/main.cpp
+++ b/main.cpp
@@ -96,9 +96,9 @@ int main(int argc, char *argv[])
             saveAndQuit = true;
         else if (optionsOK && arg == "--")
             optionsOK = false;
-        else if (filename1.isEmpty() && arg.toLower().endsWith(".pdf"))
+        else if (filename1.isEmpty())
             filename1 = arg;
-        else if (filename2.isEmpty() && arg.toLower().endsWith(".pdf"))
+        else if (filename2.isEmpty())
             filename2 = arg;
         else
             out << "unrecognized argument '" << arg << "'\n";
